addons/sale/report/sale_report.py

```diff
@@ -195,6 +197,7 @@ def _group_by_sale(self):
             p.product_tmpl_id,
             partner.country_id,
             partner.industry_id,
+            partner.zip
```

Comment:
```suggestion
            partner.zip,
```
Missing `,` here (that's why the checks are red).